src/main/webapp/js/echarts-api.js

```diff
@@ -31,7 +31,14 @@ const echartsJenkinsApi = {
      * @return {string|string}
      */
     resolveJenkinsColor: function (colorName) {
-        return getComputedStyle(document.body).getPropertyValue(colorName) || '#333';
+        const color = getComputedStyle(document.body).getPropertyValue(colorName) || '#333';
+
+        try {
+            return new Color(color).to("sRGB").toString({format: "hex"});
+        }
+        catch (e) {
+            return '#333';
```

Comment:
@janfaracik Is the relative Jenkins color `oklch(from oklch(55% 0.2308 256.91deg) 2% 0.075 h)` a valid color? I did not find a tool where I can validate such colors...

Such relative colors are not supported by colorjs.io yet. If the color is valied, then I will file a bug report in https://github.com/color-js/color.js.